tests/lcs/core/interface/interfaceui.livecodescript

```diff
@@ -37,16 +37,22 @@ get the formattedRect of char 1 of field "tf"
 local tPoint
 put (item 1 of it + item 3 of it) div 2 & "," & (item 2 of it + item 4 of it) div 2 into tPoint
 
-click at tPoint
-TestAssert "test", the clickChar is "a"
+click button 3 at tPoint
+if TestIsInStandalone() then
+   TestAssertBroken "the clickChar is a", the clickChar is "a", "not sure"
```

Comment:
What's the deal with this "not sure"? I think it's a bug if these aren't working in a standalone isn't it? In which case the reason should be the bug number. Or are you not sure if it is a bug or not?